cmd/root/collection/cluster.go

```diff
@@ -56,17 +56,73 @@ func ClusterK8sExecute(namespace string, cs CopyStrategy, ddfs helpers.Filesyste
 	return nil
 }
 
+func GetClusterLogs(namespace string, cs CopyStrategy, ddfs helpers.Filesystem, c Collector, k string, pods []string) error {
+	path, err := cs.CreatePath("kubernetes", "container-logs", "")
+	if err != nil {
+		simplelog.Errorf("trying to construct cluster config path %v with error %v", path, err)
+		return err
+	}
+
+	// Loop over dremio pods
+	for _, pod := range pods {
+		kubectlArgs := []string{k, "-n", namespace, "get", "pods", string(pod), "-o", `jsonpath={.spec['containers','initContainers'][*].name}`}
+		containers, err := clusterExecutePod(kubectlArgs)
+		if err != nil {
+			simplelog.Errorf("trying to list containers from pod %v with error %v", pod, err)
+			return err
+		}
+		// Loop over each container, construct a path and log file name
+		// write the output of the kubectl logs command to a file
+		for _, container := range s2s(containers) {
```

Comment:
skip the wrapping of s2s which is not obvious at a glance what it is, and strings.Split(containers, " ") is not hard to read either